vendor/assets/stylesheets/fenrir/building_blocks/mixins/_wrappers.scss

```diff
@@ -0,0 +1,126 @@
+
+
+
+//----------------------------------------------\\
+// Charlie slice wrapper
+//----------------------------------------------\\
+
+@mixin w-slice() {
+  padding-left: 15px;
+  padding-right: 15px;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+
+  &--no-wrap {
+    flex-wrap: nowrap;
+  }
+
+  &--no-padding {
+    padding: 0;
+  }
+}
+
+
+
+//----------------------------------------------\\
+// Charlie card wrapper
+//----------------------------------------------\\
+
+@mixin w-card() {
+  position: relative;
+  width: 100%;
+  border-radius: $br;
+  margin: 15px 0;
+  box-shadow: 0 0 0 1px color('black', A5);
+
+  transition: background-color 0.2s,
+              color 0.2s;
+
+  > :first-child {
+    border-top-left-radius: $br;
+    border-top-right-radius: $br;
+
+    &.w-slice {
+      > :first-child { border-top-left-radius: $br; }
+      > :last-child { border-top-right-radius: $br; }
+    }
+  }
+
+  > :last-child {
+    border-bottom-left-radius: $br;
+    border-bottom-right-radius: $br;
+
+    &.w-slice {
+      > :first-child { border-bottom-left-radius: $br; }
+      > :last-child { border-bottom-right-radius: $br; }
+    }
+  }
+}
+
+
+
+//----------------------------------------------\\
+// Charlie button wrapper
+//----------------------------------------------\\
+
+@mixin w-btn() {
+  // Reset button
+  background: transparent;
+  border: 0;
+  display: inline-block;
+  outline: none;
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  vertical-align: top; // make sure <button> renders inline with <a>
+  text-decoration: none;
+  text-align: left;
```

Comment:
I believe it has to do with this being just the wrapper, and not the button component. I.e. you'd usually add whatever you want in here, so the default is reset. In which case we can probably just delete the full line.

We'll review the wrapper code and talk about it before we start properly using it.